tsconfig.json

```diff
@@ -19,12 +19,12 @@
 		"noUnusedParameters": false,
 		"noUnusedLocals": false,
 
-		"sourceMap": true
+		"sourceMap": true,
 
-	},
-
-	"files": [
```

Comment:
Oh ok, I can try out the baseDir and see if IntelliJ stops freaking out. I've tried a lot of different permutations over time to keep the build happy and keep IntelliJ happy